### PR TITLE
[INJIMOB-683]: add testid for missed vc field

### PR DESCRIPTION
**Pull request by vijay151096**

### CODE REVIEW

File: components/ProfileIcon.tsx [hunk 1]
```diff
@@ -6,17 +6,25 @@ import {SvgImage} from './ui/svg';
 
 export const ProfileIcon: React.FC = props => {
   return (
-    <>
-      <View style={Theme.Styles.ProfileIconContainer}>
+    <React.Fragment>
+      <View
+        style={Theme.Styles.ProfileIconContainer}
+        testID={`ProfileIconOuter`}>
```
Comment (swatigoel) [inline components/ProfileIcon.tsx:12]: use testIDProps method available in commonUtils

File: components/ProfileIcon.tsx [hunk 2]
```diff
@@ -6,17 +6,25 @@ import {SvgImage} from './ui/svg';
...
         {props?.isPinned && SvgImage.pinIcon()}
         <View
+          testID={`ProfileIconInner`}
```
Comment (swatigoel) [inline components/ProfileIcon.tsx:15]: use testIDProps method available in commonUtils

File: components/ProfileIcon.tsx [hunk 3]
```diff
@@ -6,17 +6,25 @@ import {SvgImage} from './ui/svg';
...
           style={[
             Theme.Styles.ProfileIconInnerStyle,
             !props?.isPinned && Theme.Styles.ProfileIconPinnedStyle,
           ]}>
-          <Icon name="person" color={Theme.Colors.Icon} size={40} />
+          <Icon
+            testID={`ProfileIcon`}
```
Comment (swatigoel) [inline components/ProfileIcon.tsx:21]: use testIDProps method available in commonUtils

**Review by swatigoel — CHANGES_REQUESTED** [review]

File: components/ProfileIcon.tsx (hunk 1 above)
Comment (vijay151096) [reply components/ProfileIcon.tsx]: used the testIDProps for adding testID

File: components/ProfileIcon.tsx (hunk 3 above)
Comment (vijay151096) [reply components/ProfileIcon.tsx]: used the testIDProps for adding testID

File: components/ProfileIcon.tsx (hunk 2 above)
Comment (vijay151096) [reply components/ProfileIcon.tsx]: used the testIDProps for adding testID

**Review by swatigoel — APPROVED** [review]

**Pull request merged**